packages/electrode-archetype-react-app-dev/lib/dev-admin/middleware.js

```diff
@@ -214,6 +218,15 @@ class Middleware {
 
     const transferIsomorphicAssets = (fileSystem, cb) => {
       const isoConfig = loadIsomorphicConfig();
+      const loadableStats = `/${LOADABLE_STATS}`;
+
+      if (fileSystem.existsSync(loadableStats)) {
+        const source = fileSystem.readFileSync(loadableStats);
+        const dir = Path.resolve("./dist/js");
+        if (!Fs.existsSync(dir)) shell.mkdir("-p", dir);
+        Fs.writeFile(`${dir}${loadableStats}`, source, cb);
```

Comment:
the function `transferIsomorphicAssets` now could end up invoking the callback `cb` twice.